{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e9882e60>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание простой модели, проверка и удаление","uid":"b612aaa721c07c94","parentUid":"bff083cfa8e7f4d09de216fe041bbe96","status":"broken","time":{"start":1691061386972,"stop":1691061386972,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Создание и удаление корневого объекта","uid":"6b42fb28bcc9b338","parentUid":"bff083cfa8e7f4d09de216fe041bbe96","status":"broken","time":{"start":1691061387986,"stop":1691061387986,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"bff083cfa8e7f4d09de216fe041bbe96"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e9696b30>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание простой модели, добавление в нее компонентов, проверка и удаление","uid":"567b6013031020bd","parentUid":"e5d9e403fcfc4cb8aec4e06911a20cdd","status":"broken","time":{"start":1691061387188,"stop":1691061387190,"duration":2},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"e5d9e403fcfc4cb8aec4e06911a20cdd"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e9342bc0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание двух простых моделей с одинаковыми именами, проверка и удаление","uid":"cf038bdc982ad870","parentUid":"2f7464f8496c9639990031a1f4a38c32","status":"broken","time":{"start":1691061387397,"stop":1691061387400,"duration":3},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"2f7464f8496c9639990031a1f4a38c32"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e9202c50>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание сложной модели, содержащей две дочерние модели по каждой базовой модели, удаление одной из дочерних моделей, проверка и удаление сложной модели","uid":"bb12aabd8ac07bcf","parentUid":"4fe8684d69f7838e6e002a4d2def9f12","status":"broken","time":{"start":1691061387588,"stop":1691061387591,"duration":3},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"4fe8684d69f7838e6e002a4d2def9f12"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e91e3700>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание сложной моедели с двумя дочерними моделями с одинаковыми именами, проверка и удаление","uid":"fb42973ce78bbc14","parentUid":"495a16f6eb2b1c43f77831747ca93135","status":"broken","time":{"start":1691061387796,"stop":1691061387798,"duration":2},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"495a16f6eb2b1c43f77831747ca93135"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e931a4a0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание двух корневых объектов с одинаковыми именами, проверка и удаление","uid":"7e46cc4d0afa9564","parentUid":"f53166900ea91aaa61d49ca62186c07b","status":"broken","time":{"start":1691061388179,"stop":1691061388182,"duration":3},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"f53166900ea91aaa61d49ca62186c07b"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e90b72b0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание корневого объекта, добавление в него объекта по модели Directory, проверка и удаление корневого объекта","uid":"cb166a61253c540a","parentUid":"435bd0d303d149e0596fdaec72d7a6eb","status":"broken","time":{"start":1691061388385,"stop":1691061388387,"duration":2},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"435bd0d303d149e0596fdaec72d7a6eb"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e957eb60>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание корневого объекта, добавление в него объекта по модели Directory, добавление второго объекта по модели Directory с тем же именем, проверка и удаление корневого объекта","uid":"5f70938825523166","parentUid":"657728ab96410f0fe524371b4ade7a45","status":"broken","time":{"start":1691061388611,"stop":1691061388613,"duration":2},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"657728ab96410f0fe524371b4ade7a45"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e9347220>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание корневого объекта, добавление в него двух объектов по модели Directory с уникальными именами. Удаление одного дочернего объекта, проверка. Удаление корневого объекта","uid":"4870122d4bf1a811","parentUid":"16df6dda9ee765d5e31b8974be6f11a2","status":"broken","time":{"start":1691061388818,"stop":1691061388821,"duration":3},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"16df6dda9ee765d5e31b8974be6f11a2"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e916ed70>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание модели, содержащей подмодели. Создание корневого объекта по модели Directory. Добавление в него объекта по созданной модели. Проверка и удаление объекта. Удаление модели.","uid":"7210bbf8bd4b4f95","parentUid":"fefbe4f0db5d525ea675ad3f5ea998a9","status":"broken","time":{"start":1691061389046,"stop":1691061389049,"duration":3},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"fefbe4f0db5d525ea675ad3f5ea998a9"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e933b4f0>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание модели, содержаей подмодели. Создание корневого объекта по модели Directory. Добавление в него объекта по созданой модели. Проверка объекта. Удаление модели с удалением объекта. Проверка факта удаления объекта. Удаление корневого объекта.","uid":"871f85178da4ca13","parentUid":"96967f4bd736410ece93116501944fca","status":"broken","time":{"start":1691061389281,"stop":1691061389283,"duration":2},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"96967f4bd736410ece93116501944fca"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e932eb30>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"Создание модели, содержащей подмодели. Создание корневого объекта по модели Directory. Добавление в него объекта по созданной модели. Проверка объекта. Удаление модели с отвязкой объекта. Проверка факта сохранения объекта и изменения типа модели на Directory. Удаление корневого объекта.","uid":"dcd4ea15587353a","parentUid":"800d9543d3721c6186dbedd8c402e0b1","status":"broken","time":{"start":1691061389486,"stop":1691061389489,"duration":3},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"800d9543d3721c6186dbedd8c402e0b1"},{"name":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=49359): Max retries exceeded with url: /session/7c8aec4adb4fb396c89f763efe720e8e/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e90bfa60>: Failed to establish a new connection: [Errno 111] Connection refused'))","children":[{"name":"test_platform","uid":"6b26f3a1dd619797","parentUid":"2b62d8dde49928e8dee4ce4ff47c223c","status":"broken","time":{"start":1691061389719,"stop":1691061389721,"duration":2},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"2b62d8dde49928e8dee4ce4ff47c223c"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}